internal/authorization/authorizer_test.go

```diff
@@ -194,8 +194,34 @@ func (s *AuthorizerSuite) TestShouldCheckGroupMatching() {
 		}).
 		Build()
 
-	tester.CheckAuthorizations(s.T(), John, "https://protected.example.com/", Bypass)
-	tester.CheckAuthorizations(s.T(), Bob, "https://protected.example.com/", Denied)
+	tester.CheckAuthorizations(s.T(), John, "https://protected.example.com/", []byte("GET"), Bypass)
+	tester.CheckAuthorizations(s.T(), Bob, "https://protected.example.com/", []byte("GET"), Denied)
+}
```

Comment:
Can you also add a test checking the authorization module works when no `X-Forwarded-Method` is provided, meaning CheckAuthorizations is called with `nil` instead of `[]byte("GET")`.